### PR TITLE
Remove parallax building hours image

### LINKED ISSUE #1395: Landscape building hours images are not wide enough
They do not expand to the full width of the view. Calculation bug with the parallax view?

### PR DESCRIPTION
**Pull request by hawkrives**
(TBQF, I've never liked the parallax effect in building hours, and I've experienced it being slow several times. With that said, I think I have reasonable reasons for wanting to remove it, as I will attempt to outline below.)

1. **The library appears to be unmaintained**
      The last commit is from September 2016, and https://github.com/lelandrichardson/react-native-parallax-view/pulls has several bugfixes, including one for #1395.

2. **The library has bugs**
      see #1395

3. okay, I don't have a third reason.

This PR is not perfect; my packager is currently down, so I can't check my changes, but I know that the parallax view at least added an inset box-shadow to the image, which I have not replicated here. I would, of course, add that shadow before merging this.

TL;DR: I want to remove `react-native-parallax-view`.

Closes #1395

No Image | Image
--|--
screen shot 2017-09-12 at 11 46 12 pm | screen shot 2017-09-12 at 11 46 02 pm

### CODE REVIEW

**Review by drewvolz — CHANGES_REQUESTED** [review]
1. The `<Image>` view should load if there is an image, otherwise it should hide.
2. The images are zooming _way_ in and it's impossible to tell what the images are of.
3. Like you mentioned, inset box-shadow would be great.

**Comment by hawkrives** [comment]
I'm going to push back on (1), since that's not the current behavior. The other two I'll take on.

**Comment by drewvolz** [comment]
(1) is the current behavior, I think. It's displaying a blank placeholder in the _current_, and _this_ version would just make that placeholder a lot of extra space without a placeholder?

**Comment by hawkrives** [comment]
Ah. I interpreted (1) as "don't show anything if there is no image" - I quite like the current behavior of maintaining the same layout regardless of image status by way of using some kind of placeholder image.

**Comment by drewvolz** [comment]
With these changes that do not add anything in the blank space, I will say it feels off.

**Comment by drewvolz** [comment]
Your interpretation of what I said was correct. If you want a placeholder, that's fine. But then we should add one to this PR if that's the way this is going.

**Comment by hawkrives** [comment]
My end goal here is to replicate the current status of this view, merely without the parallax effect. I will commence work on this once my packager deigns to work with me.

**Comment by drewvolz** [comment]
@hawkrives ping

**Comment by hawkrives** [comment]
Not gonna make it in 2.4

**Review by drewvolz — APPROVED** [review]
We fixed the image resizing and fixed up the blank image space issue.

...and dropped the inset box shadow.

**Comment by hawkrives** [comment]
I approve. 

@drewvolz can you add screenshots of the standard and missing image views?

**Comment by drewvolz** [comment]
Updated.

**Comment by hawkrives** [comment]
Still approved from me. 

@rye or @elijahverdoorn do either of you want to approve this or look at it before it gets merged?

**Review by rye — APPROVED** [review]

**Review by elijahverdoorn — APPROVED** [review]
I never experience the parallax on my device, since all of the hours fit on the screen. With that said, I like the effect overall, but not at the cost of performance. I trust @hawkrives and @drewvolz, and if you guys say it should go, it should go. 😄
